package.json

```diff
@@ -26,6 +26,7 @@
     "d3-transition": "1.2.0",
     "dns": "^0.2.2",
     "mapbox-gl": "^1.12.0",
+    "npm": "^6.14.8",
```

Comment:
what is this used for?

Reply:
no idea... npm said it needed an update so I said yes, but I'm not sure why it did it inside the repo